packages/myst-cli-utils/src/utils.ts

```diff
@@ -19,6 +19,9 @@ export function clirun<S extends ISession>(
     program: Command;
     getSession: (logger: Logger, opts?: SessionOpts) => S;
   },
+  runOptions?: {
+    forceExit?: boolean | ((...args: any[]) => boolean);
```

Comment:
Maybe we should flip this to have this be the default and then processes that need to hang (e.g. start/watch) can add work arounds.